syncto/tests/test_functional.py

```diff
@@ -136,3 +136,7 @@ def setUp(self):
     def test_record_handle_cors_headers(self):
         resp = self.app.get(RECORD_URL, headers=self.headers, status=200)
         self.assertIn('Access-Control-Allow-Origin', resp.headers)
+
+    def test_can_delete_record(self):
+        self.sync_client.return_value.delete_record.return_value = None
+        self.app.delete(RECORD_URL, headers=self.headers, status=204)
```

Comment:
- What happens if we try to delete an unexisting resource?
- What happens if we try to delete a resource which we don't own?